gp-nested-forms/gpnf-hide-expired-nested-form-field.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Gravity Perks // GP Nested Forms // Hide Nested Form Field if the Nested Form has expired.
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * Instruction Video: https://www.loom.com/share/798f30d73d254864aa6a65e7d16d390b
+ */
+add_filter( 'gform_pre_render', 'check_nested_form_expiration' );
+function check_nested_form_expiration( $form ) {
```

Comment:
If you keep this as a function and not a class, I'd either prefix it or go with an anonymous function.